<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_page</class>
 <widget class="QWidget" name="main_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="verticalGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>450</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnAddAccount">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 290 9pt &quot;微软雅黑 Light&quot;;
    border: none; /* 控件本身无边框 */
    color: #000000; /* 文字颜色 */
    font-weight: normal;
    padding: 5px; /* 按需调整内边距 */
}

QPushButton:hover {
    text-decoration: underline; /* 鼠标悬浮文字加下划线 */
    color: #1e92ff; /* 文字颜色变浅 */
}

QPushButton:pressed {
    border-style: solid; /* 按下时边框样式 */
    border-width: 1px; /* 按下时边框宽度 */
    border-color: #8c8884; /* 按下时边框颜色 */
    border-radius: 5px; /* 边框圆角 */
}
</string>
      </property>
      <property name="text">
       <string>新增账目</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnManageAccount">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 290 9pt &quot;微软雅黑 Light&quot;;
    border: none; /* 控件本身无边框 */
    color: #000000; /* 文字颜色 */
    font-weight: normal;
    padding: 5px; /* 按需调整内边距 */
}

QPushButton:hover {
    text-decoration: underline; /* 鼠标悬浮文字加下划线 */
    color: #1e92ff; /* 文字颜色变浅 */
}

QPushButton:pressed {
    border-style: solid; /* 按下时边框样式 */
    border-width: 1px; /* 按下时边框宽度 */
    border-color: #8c8884; /* 按下时边框颜色 */
    border-radius: 5px; /* 边框圆角 */
}
</string>
      </property>
      <property name="text">
       <string>管理账目</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDataAnalysis">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 290 9pt &quot;微软雅黑 Light&quot;;
    border: none; /* 控件本身无边框 */
    color: #000000; /* 文字颜色 */
    font-weight: normal;
    padding: 5px; /* 按需调整内边距 */
}

QPushButton:hover {
    text-decoration: underline; /* 鼠标悬浮文字加下划线 */
    color: #1e92ff; /* 文字颜色变浅 */
}

QPushButton:pressed {
    border-style: solid; /* 按下时边框样式 */
    border-width: 1px; /* 按下时边框宽度 */
    border-color: #8c8884; /* 按下时边框颜色 */
    border-radius: 5px; /* 边框圆角 */
}
</string>
      </property>
      <property name="text">
       <string>数据分析</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnUserHelp">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 290 9pt &quot;微软雅黑 Light&quot;;
    border: none; /* 控件本身无边框 */
    color: #000000; /* 文字颜色 */
    font-weight: normal;
    padding: 5px; /* 按需调整内边距 */
}

QPushButton:hover {
    text-decoration: underline; /* 鼠标悬浮文字加下划线 */
    color: #1e92ff; /* 文字颜色变浅 */
}

QPushButton:pressed {
    border-style: solid; /* 按下时边框样式 */
    border-width: 1px; /* 按下时边框宽度 */
    border-color: #8c8884; /* 按下时边框颜色 */
    border-radius: 5px; /* 边框圆角 */
}
</string>
      </property>
      <property name="text">
       <string>用户帮助</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSettings">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 290 9pt &quot;微软雅黑 Light&quot;;
    border: none; /* 控件本身无边框 */
    color: #000000; /* 文字颜色 */
    font-weight: normal;
    padding: 5px; /* 按需调整内边距 */
}

QPushButton:hover {
    text-decoration: underline; /* 鼠标悬浮文字加下划线 */
    color: #1e92ff; /* 文字颜色变浅 */
}

QPushButton:pressed {
    border-style: solid; /* 按下时边框样式 */
    border-width: 1px; /* 按下时边框宽度 */
    border-color: #8c8884; /* 按下时边框颜色 */
    border-radius: 5px; /* 边框圆角 */
}
</string>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
